### PR TITLE
[Android/Api] method to check element availability

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add new method to check element available on the pipeline.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

File: api/android/api/src/main/java/org/nnsuite/nnstreamer/Pipeline.java [hunk 1]
```diff
@@ -157,6 +158,23 @@ public Pipeline(@NonNull String description, @Nullable StateChangeCallback callb
...
+     * Checks the element is registered and available on the pipeline.
+     *
+     * @param element The name of GStreamer element
+     *
+     * @return true if the element is available
+     *
+     * @throws IllegalArgumentException if given param is invalid
+     */
+    public static boolean isElementAvailable(@NonNull String element) {
+        if (element == null || element.isEmpty()) {
```
Comment (dongju-chae) [inline api/android/api/src/main/java/org/nnsuite/nnstreamer/Pipeline.java:171]: I think this null check is not necessary as `element` is annotated as `@NonNull`

Comment (jaeyun-jung) [reply api/android/api/src/main/java/org/nnsuite/nnstreamer/Pipeline.java:171]: NonNull annotation is just warning message in android-studio. This does not guarantee the passed param is not null.

Comment (dongju-chae) [reply api/android/api/src/main/java/org/nnsuite/nnstreamer/Pipeline.java:171]: Yes.. it's the caller's responsibility.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**